lib/src/google_navigation_flutter_platform_interface.dart

```diff
@@ -572,6 +573,215 @@ abstract mixin class MapViewAPIInterface {
   }
 }
 
+abstract mixin class AutoMapViewAPIInterface {
+  /// Awaits the platform view to be ready for communication.
```

Comment:
Awaits the **auto** view to be ready for communication

Have you tested if this method works?

Reply:
Nope. I'll add something to example app.